### DISCUSSION TITLE
上記リンクにあるFLASHボタンとは？

### DISCUSSION THREAD

**Discussion by willelz** [Yowkees/keyball#123] [Q&A] [answered]
keyball39/61の`3-7．ProMicoの書き込みと動作確認`に書かれている`上記リンクにあるFLASHボタン`がどこのことかわかりません。
REMAPの設定画面のことでしょうか。

**Comment by Yowkees** [comment, marked as answer]
![FLASH](https://user-images.githubusercontent.com/84209772/168497521-c8d9cd2c-39ee-41bb-b7c6-70744b906801.jpg)
ビルドガイドに示したリンクにアクセスすると、この画像のような画面に移動すると思います。
この赤丸部のFLASHボタンを押下後に順に進んでいくと、書き込みが開始されると思います。
書き込みエラーとなる場合には、USBデバイス選択の直前にリセットボタンを2回すばやく押してみてください。

**Reply by willelz** [reply]
PCのブラウザで再度確認したところボタンが表示されました。
ありがとうございました。
